nibabies/workflows/anatomical/base.py

```diff
@@ -196,32 +199,37 @@ def init_infant_anat_wf(
     if skull_strip_mode != "force":
         raise NotImplementedError("Skull stripping is currently required.")
 
-    brain_extraction_wf = init_infant_brain_extraction_wf(
-        age_months=age_months,
-        ants_affine_init=ants_affine_init,
-        skull_strip_template=skull_strip_template.space,
-        template_specs=skull_strip_template.spec,
-        omp_nthreads=omp_nthreads,
-        sloppy=sloppy,
-        debug="registration" in config.execution.debug,
-    )
+    if precomp_mask:
+        precomp_mask_wf = init_precomputed_mask_wf(omp_nthreads=omp_nthreads)
+        precomp_mask_wf.inputs.inputnode.t1w_mask = precomp_mask
 
-    coregistration_wf = init_coregistration_wf(
-        omp_nthreads=omp_nthreads,
-        sloppy=sloppy,
-        debug="registration" in config.execution.debug,
-    )
-    coreg_report_wf = init_coreg_report_wf(
-        output_dir=output_dir,
-    )
+    else:
+        brain_extraction_wf = init_infant_brain_extraction_wf(
+            age_months=age_months,
+            ants_affine_init=ants_affine_init,
+            skull_strip_template=skull_strip_template.space,
+            template_specs=skull_strip_template.spec,
+            omp_nthreads=omp_nthreads,
+            sloppy=sloppy,
+            debug="registration" in config.execution.debug,
+        )
+        coregistration_wf = init_coregistration_wf(
+            omp_nthreads=omp_nthreads,
+            sloppy=sloppy,
+            debug="registration" in config.execution.debug,
+        )
+        coreg_report_wf = init_coreg_report_wf(
+            output_dir=output_dir,
+        )
```

Comment:
since the T2w is only being used to generate the mask, nope. I was actually planning on removing the T2w restriction if a precomputed mask is available.

Reply:
Are we not outputting the coregistered T2w image? I think that's been a desired feature for fMRIPrep for a while, if it hasn't been explicitly brought up here.

Reply:
no we are not (yet)